sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/OfferQueueTest.java

```diff
@@ -0,0 +1,132 @@
+package com.mesosphere.sdk.scheduler;
+
+import com.mesosphere.sdk.testutils.TestConstants;
+import org.apache.mesos.Protos;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This class tests the {@link OfferQueue}.
+ */
+public class OfferQueueTest {
+    @Test
+    public void testEmptyQueue() {
+        OfferQueue offerQueue = new OfferQueue();
+        Assert.assertTrue(offerQueue.isEmpty());
+    }
+
+    @Test
+    public void testEnqueueOffer() {
+        OfferQueue offerQueue = new OfferQueue();
```

Comment:
Instead of requiring access to OfferQueue.DEFAULT_CAPACITY, these tests could define their own TEST_CAPACITY and pass that to the OfferQueue constructor. Then DEFAULT_CAPACITY could remain `private`